src/components/SamplePanel/index.js

```diff
@@ -74,6 +75,13 @@ const SamplePanel = ({sampleName, sampleData, config, reference, socket, panelEx
   const sampleColours = {}; /* dataformat needed by <CoveragePlot> */
   sampleColours[sampleName] = sampleColour;
 
+  var variant_name="";
```

Comment:
here we again can prefer `.map.join()`. Also, `var` is not good, prefer `const/let`